rules/opentitan.bzl

```diff
@@ -4,14 +4,14 @@
 
 load("@rules_cc//cc:find_cc_toolchain.bzl", "find_cc_toolchain")
 load(
-    "//rules:cc_side_outputs.bzl",
+    "@lowrisc_opentitan//rules:cc_side_outputs.bzl",
```

Comment:
Within our repo, I _think_ `@lowrisc_opentitan//` can be abbreviated as just `@//`.

See https://bazel.build/concepts/labels.

This is a more general tech-debt problem.  Would you mind filing an issue to clean up bazel labels within the repo?

Reply:
We also need to consider whether or not another project might import us as a dependency into their project (but not in this PR).